src/main/resources/anniv5Resources/summaries/tags.txt

```diff
@@ -11,6 +11,7 @@ BellordPack	None
 BitingColdPack	Debuffs
 BlacksmithsPack	None
 BoardGamePack	None
+BulwarkPack	Exhaust
```

Comment:
Feel No Pain is strong enough support that I put the Exhaust tag on the pack. Could remove it if we think it's misleading, but even if the other 9 cards didn't exist, adding FNP to the card pool would still be one of the strongest things you can do when playing with packs like Distortion or Psychic.